src/components/formio/datamap.tsx

```diff
@@ -8,12 +9,19 @@ export interface DataMapProps {
   name: string;
   valueComponent: React.ReactElement;
   keyLabel: React.ReactNode;
+  ariaLabelMessage?: MessageDescriptor;
 }
 
 /**
  * Data map with readonly keys.
  */
-export const DataMap: React.FC<DataMapProps> = ({name, valueComponent, keyLabel = 'Key'}) => {
+export const DataMap: React.FC<DataMapProps> = ({
+  name,
+  valueComponent,
+  keyLabel = 'Key',
+  ariaLabelMessage = null,
```

Comment:
```suggestion
  ariaLabelMessage,
```
I think you can just leave it as `undefined` - that's also falsy